Doppler.PushContact/Controllers/PushContactController.cs

```diff
@@ -132,5 +132,19 @@ public async Task<IActionResult> GetMessageDetails([FromRoute] string domain, [F
                 messageDetails.NotDelivered
             });
         }
+
+        [HttpGet]
+        [Route("push-contacts/domains")]
+        public async Task<IActionResult> GetQuantityByDomain()
```

Comment:
It is not clear the relation beweed the URL and the method name.

What do you think about return domain details in place of a quantity?

```suggestion
        [Route("push-contacts/domains")]
        public async Task<IActionResult> GetDomains()
```

Reply:
I have to return only the domain name and quantity of push contacts associated with it, not all the information. I think that "GetDomains" name is not the best option.

Reply:
If `GetDomains` is not the best option, probably the URL `push-contacts/domains` is not a good option either.